apps/ml/site/project/projects.jag

```diff
@@ -595,12 +609,30 @@ include("../../includes/tenantAware.jag");
 
     // function to append button disabled CSS class when there are no analyses in a project
     function appendDisabled(noOfAnalyses) {
+
         var classAppend = '';
         if(noOfAnalyses == 0) {
             classAppend = 'btn-disabled';  
         }
         return classAppend;
     }
+    // function to append button disabled CSS class when there are no complete models in a project

```

Comment:
Use proper linguistics for method comments. Use capitals to begin the sentences.